pgcontents/query.py

```diff
@@ -700,6 +731,68 @@ def purge_remote_checkpoints(db, user_id):
     )
 
 
+def analyze_checkpoints(engine, crypto_factory, min_dt=None, max_dt=None):
```

Comment:
Can we rename all these functions to something like `generate_checkpoints`/`generate_files`? Doesn't looks like they do any analysis.